src/__tests__/perseus-markdown-test.jsx

```diff
@@ -63,8 +63,10 @@ var htmlFromMarkdown = function(source) {
 var assertParsesToReact = function(source, html) {
     var actualHtml = htmlFromMarkdown(source);
     if (actualHtml !== html) {
+        /* eslint-disable no-console */
```

Comment:
There's a [TODO about that in khan-linter](https://github.com/Khan/khan-linter/blob/master/linters.py#L478) so it looks like that's not implemented there yet.